templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete a manufacturer</h1>
+  <p>Are you sure you want to delete a manufacturer: {{ manufacturer }}?</p>
+  <p><i>All cars from this manufacturer will be also deleted!</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger"/>
+    <button class="btn btn-success">
+      <a class="btn btn-success" href="{% url 'taxi:manufacturer-list' %}" style="color: white;">
+        Cancel
+      </a>
+    </button>

```

Comment:
The 'Cancel' button should not be wrapped inside a button tag when it already uses an anchor tag for navigation. You can directly use the anchor tag with the appropriate classes for styling.